src/Native/Runtime/amd64/ExceptionHandling.asm

```diff
@@ -383,6 +383,11 @@ endif
         movdqa  xmm15,[r8 + OFFSETOF__REGDISPLAY__Xmm + 9*10h]
 
         mov     rcx, [rsp + rsp_offsetof_arguments + 0h]            ;; rcx <- exception object
+
+ifdef CORERT
+        mov     rdx, rcx                                            ;; RyuJIT expects the exception object in rdx
```

Comment:
(On a second thought I could have just loaded directly into `rdx`, but I would hope this code will only be temporary even if it happens to be the right fix.)